packages/syft/src/syft/service/sync/sync_state.py

```diff
@@ -294,4 +295,7 @@ def _repr_html_(self) -> str:
             {date_html}
         </div>
 """
-        return repr + self.rows._repr_html_()
+
+        rews = get_repr_values_table(self.rows, True)
+        return repr + create_table_template(rews, "SyncStateRow", grid_template_columns="auto auto 1fr auto", grid_template_cell_columns="unset", table_icon=None)
```

Comment:
Can we move this to the class we're tabulating instead, and handle it where we construct the table? Something like:
```
class SyncStateRow(SyftObject):
    __table_coll_widths__: ClassVar[list[str] | None] = ['auto', 'auto', '1fr', 'auto']
    ...
```